src/geometry/ArborX_HyperBox.hpp

```diff
@@ -27,7 +27,7 @@ namespace ArborX::ExperimentalHyperGeometry
  * size 2x spatial dimension with a default constructor to initialize
  * properly an "empty" box.
  */
-template <int DIM, class FloatingPoint = float>
+template <int DIM, class Coordinate = float>
```

Comment:
I would prefer
```suggestion
template <int DIM, class CoordinateType = float>
```
to make clear that this is a type, not a value.

Reply:
I'm of two minds about it. I thought about it, but decided to go with `Coordinate`. Similar to `BVH`'s `Value` instead of `ValueType`.

Reply:
It's probably not worth holding up this pull request over it. Changing those would be backward-compatible anyway.